bof-request.md

```diff
@@ -0,0 +1,43 @@
+# Name: Media over QUIC
+## Description
+ 
+Media over QUIC (moq) will work on solution for a simple low-latency 
+media delivery protocol that is common across ingest and distribution. 
```

Comment:
I would remove the words "common across" as that seems like it might just trigger people but I can go either way.